File: ResourceManager/Helpers/FilterParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ResourceManager.Helpers
{
    /// <summary>  
    /// Filter parameters Model Class  
    /// </summary>  
    public class FilterParams
    {
        public string ColumnName { get; set; } = string.Empty;
        public string FilterValue { get; set; } = string.Empty;
        public FilterOptions FilterOption { get; set; } = FilterOptions.Contains;
    }
}

File: ResourceManager/Helpers/FilterParams.vb

Namespace ResourceManager.Helpers
    ''' <summary>  
    ''' Filter parameters Model Class  
    ''' </summary>  
    Public Class FilterParams
        Public Property ColumnName As String = String.Empty
        Public Property FilterValue As String = String.Empty
        Public Property FilterOption As FilterOptions = FilterOptions.Contains
    End Class
End Namespace
